internal/conf/configuration.go

```diff
@@ -490,6 +497,19 @@ func (t *TextlocalProviderConfiguration) Validate() error {
 	return nil
 }
 
+func (t *Msg91ProviderConfiguration) Validate() error {
+	if t.AuthKey == "" {
+		return errors.New("missing Msg91 Auth key")
+	}
+	if t.DltTemplateId == "" {
+		return errors.New("missing Msg91 Template Id")
+	}
+	if t.SenderId == "" {
+		return errors.New("missing Msg91 Sender Id")
+	}
+	return nil
+}
```

Comment:
It would be helpful, if possible, to check the format of these values.

Reply:
Can you explain, like should i check for string type ?. Sorry i didnt understand